docker-compose.yml

```diff
@@ -446,6 +452,9 @@ services:
       HOP_SERVER_PASS: ${HOP_SERVER_PASS}
       HOP_SERVER_PORT: 8080
       HOP_SERVER_HOSTNAME: 0.0.0.0
+      HOP_MINIO_ACCES: minio
+      HOP_MINIO_SECRET: miniorepan
+
```

Comment:
why exposing this credentials? anyone can login with these credentials to minio and delete files

Reply:
Yes I Forgot this when I was testing